common/changes/office-ui-fabric-react/wiolsen-pr-changes_2019-06-18-22-39.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Updates to tileslist, list and marquee selection",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

This should be minor since we're adding backwards compatible API